wp-content/themes/theme_name/template-parts/header/head.php

```diff
@@ -0,0 +1,17 @@
+<meta charset="<?php bloginfo( 'charset' ); ?>" />
+<meta content="width=device-width, initial-scale=1.0" name="viewport">
+<meta http-equiv="Content-type" content="text/html; charset=utf-8">
+<meta name="keywords" content="" />
```

Comment:
We can remove keywords, they are not that SEO important anymore, plus SEO is handled by the plugin (yoast etc.).